scipy/optimize/_linprog_highs.py

```diff
@@ -131,40 +132,54 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
         to be disabled.
     dual_feasibility_tolerance : double
         Dual feasibility tolerance.  Default is 1e-07.
-        This option is ignored when ``solver='ipm'``.
+        The minimum of this and ``primal_feasibility_tolerance``
+        is used for the feasibility tolerance when ``solver='ipm'``.
     dual_objective_value_upper_bound : double
         Upper bound on objective value for dual simplex:
         algorithm terminates if reached.  Default is the largest possible
         value for a ``double`` on the platform.
         When ``solver='ipm'`` this value is ignored.
+    ipm_optimality_tolerance : double
+        Optimality tolerance for ``solver='ipm'``.  Default is 1e-08.
+        Minimum possible value is 1e-12 and must be smaller than the largest
+        possible value for a ``double`` on the platform.
     message_level : int {0, 1, 2, 3, 4, 5, 6, 7}
         Verbosity level, corresponds to:
 
-            ``0``: No messages
+            ``0``: None
+                All messaging to stdout is supressed.
 
-            ``1``: Verbose messaging
+            ``1``: Verbose
+                Includes a once-per-iteration report on simplex/ipm
+                progress and information about each nonzero row and
```

Comment:
nonzero row and column of what?

Reply:
The LP model from what I can tell from reading their docs -- which may be misleading to scipy users because HiGHS represents the LP with row and column constraints